src/plays/web3-staking-dapp/artifacts/build-info/5faf9c6c2f8753632f3c7c13937cd196.json

```diff
@@ -0,0 +1 @@
OP PUSH2 0x11E9 PUSH1 0x0 DUP4 ADD DUP5 PUSH2 0x1041 JUMP JUMPDEST SWAP3 SWAP2 POP POP JUMP JUMPDEST PUSH1 0x0 PUSH1 0x20 DUP3 ADD SWAP1 POP PUSH2 0x1204 PUSH1 0x0 DUP4 ADD DUP5 PUSH2 0x10F2 JUMP JUMPDEST SWAP3 SWAP2 POP POP JUMP JUMPDEST PUSH1 0x0 PUSH2 0x100 DUP3 ADD SWAP1 POP PUSH2 0x1220 PUSH1 0x0 DUP4 ADD DUP12 PUSH2 0x10F2 JUMP JUMPDEST PUSH2 0x122D PUSH1 0x20 DUP4 ADD DUP11 PUSH2 0xE76 JUMP JUMPDEST PUSH2 0x123A PUSH1 0x40 DUP4 ADD DUP10 PUSH2 0x10F2 JUMP JUMPDEST PUSH2 0x1247 PUSH1 0x60 DUP4 ADD DUP9 PUSH2 0x10F2 JUMP JUMPDEST PUSH2 0x1254 PUSH1 0x80 DUP4 ADD DUP8 PUSH2 0x10F2 JUMP JUMPDEST PUSH2 0x1261 PUSH1 0xA0 DUP4 ADD DUP7 PUSH2 0x10F2 JUMP JUMPDEST PUSH2 0x126E PUSH1 0xC0 DUP4 ADD DUP6 PUSH2 0x10F2 JUMP JUMPDEST PUSH2 0x127B PUSH1 0xE0 DUP4 ADD DUP5 PUSH2 0xEF2 JUMP JUMPDEST SWAP10 SWAP9 POP POP POP POP POP POP POP POP POP JUMP JUMPDEST PUSH1 0x0 DUP2 SWAP1 POP PUSH1 0x20 DUP3 ADD SWAP1 POP SWAP2 SWAP1 POP JUMP JUMPDEST PUSH1 0x0 DUP2 MLOAD SWAP1 POP SWAP2 SWAP1 POP JUMP JUMPDEST PUSH1 0x0 PUSH1 0x20 DUP3 ADD SWAP1 POP SWAP2 SWAP1 POP JUMP JUMPDEST PUSH1 0x0 DUP3 DUP3 MSTORE PUSH1 0x20 DUP3 ADD SWAP1 POP SWAP3 SWAP2 POP POP JUMP JUMPDEST PUSH1 0x0 DUP2 SWAP1 POP SWAP3 SWAP2 POP POP JUMP JUMPDEST PUSH1 0x0 DUP3 DUP3 MSTORE PUSH1 0x20 DUP3 ADD SWAP1 POP SWAP3 SWAP2 POP POP JUMP JUMPDEST PUSH1 0x0 PUSH2 0x12E8 DUP3 PUSH2 0x13FC JUMP JUMPDEST SWAP2 POP PUSH2 0x12F3 DUP4 PUSH2 0x13FC JUMP JUMPDEST SWAP3 POP DUP3 PUSH32 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF SUB DUP3 GT ISZERO PUSH2 0x1328 JUMPI PUSH2 0x1327 PUSH2 0x144F JUMP JUMPDEST JUMPDEST DUP3 DUP3 ADD SWAP1 POP SWAP3 SWAP2 POP POP JUMP JUMPDEST PUSH1 0x0 PUSH2 0x133E DUP3 PUSH2 0x13FC JUMP JUMPDEST SWAP2 POP PUSH2 0x1349 DUP4 PUSH2 0x13FC JUMP JUMPDEST SWAP3 POP DUP3 PUSH2 0x1359 JUMPI PUSH2 0x1358 PUSH2 0x147E JUMP JUMPDEST JUMPDEST DUP3 DUP3 DIV SWAP1 POP SWAP3 SWAP2 POP POP JUMP JUMPDEST PUSH1 0x0 PUSH2 0x136F DUP3 PUSH2 0x13FC JUMP JUMPDEST SWAP2 POP PUSH2 0x137A DUP4 PUSH2 0x13FC JUMP JUMPDEST SWAP3 POP DUP2 PUSH32 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF DIV DUP4 GT DUP3 ISZERO ISZERO AND ISZERO PUSH2 0x13B3 JUMPI PUSH2 0x13B2 PUSH2 0x144F JUMP JUMPDEST JUMPDEST DUP3 DUP3 MUL SWAP1 POP SWAP3 SWAP2 POP POP JUMP JUMPDEST PUSH1 0x0 PUSH2 0x13C9 DUP3 PUSH2 0x13DC JUMP JUMPDEST SWAP1 POP SWAP2 SWAP1 POP JUMP JUMPDEST PUSH1 0x0 DUP2 ISZERO ISZERO SWAP1 POP SWAP2 SWAP1 POP JUMP JUMPDEST PUSH1 0x0 PUSH20 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF DUP3 AND SWAP1 POP SWAP2 SWAP1 POP JUMP JUMPDEST PUSH1 0x0 DUP2 SWAP1 POP SWAP2 SWAP1 POP JUMP JUMPDEST PUSH1 0x0 PUSH2 0x1411 DUP3 PUSH2 0x13FC JUMP JUMPDEST SWAP2 POP PUSH32 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF DUP3 EQ ISZERO PUSH2 0x1444 JUMPI PUSH2 0x1443 PUSH2 0x144F JUMP JUMPDEST JUMPDEST PUSH1 0x1 DUP3 ADD SWAP1 POP SWAP2 SWAP1 POP JUMP JUMPDEST PUSH32 0x4E487B7100000000000000000000000000000000000000000000000000000000 PUSH1 0x0 MSTORE PUSH1 0x11 PUSH1 0
```

Comment:
Is this file intended?

Reply:
I think it will be better if we keep this file